book/website/communication/citable/citable-linking.md

```diff
@@ -62,6 +66,440 @@ Whichever route you decide to take, it is important to double-check the proof of
+
+When you "add the DOI of the publication to the dataset" (see {ref}`Reserve DOI<cm-citable-linking-reserve-doi>`), you're creating connection metadata.
+But where exactly does this information go, and how do you create these links?
+This section provides practical guidance on using connection metadata to build a connected scholarly record.
+
+For background on persistent identifiers and the infrastructure that makes these connections possible, see our {ref}`chapter on PIDs<rr-rdm-pid>`.
+
+(cm-citable-linking-related)=
+## Linking Research Outputs
+
+When you have multiple related research outputs (a dataset, the code that analyzes it, the paper that describes it), you can formally link them in the PID metadata, for example, in DataCite metadata schema, this information is captured in the **relatedIdentifier** field.
+
+### Why Link Through Metadata?
+
+You might wonder: isn't it enough to mention the DOI in the paper text or README file?
+While that's helpful, formal metadata links provide additional benefits:
+
+- **Machine-readable**: Automated systems can discover all outputs related to a paper or dataset
+- **Bidirectional**: Both resources show the connection, not just one
+- **Typed relationships**: The metadata specifies what kind of relationship exists
+- **Aggregatable**: Citation databases and metrics systems can trace the full impact
+- **Discoverable**: Finding one output helps users discover all related outputs
+
+### Types of Relationships
+
+PID metadata schema uses standardized **relationship types** to describe how resources relate to each other.
+Examples from the DataCite schema include:
+
+| Relationship Types | Use Case | Example |
+|-------------------|----------|---------|
+| IsSupplementTo / IsSupplementedBy | When one output supplements another | A dataset *IsSupplementTo* the paper that describes it |
+| IsDerivedFrom / IsSourceOf | When one output is derived from another | A cleaned dataset *IsDerivedFrom* the raw data |
+| Cites / IsCitedBy | Formal citation relationships | A paper *Cites* a methodology it uses |
+| IsPartOf / HasPart | When outputs are components of a larger whole | Individual datasets *IsPartOf* a data collection |
+| References / IsReferencedBy | General references | Documentation *References* a protocol |
+| IsVersionOf / HasVersion | Connections between versions | (see next section) |
+| IsNewVersionOf / IsPreviousVersionOf | Specific version succession | Version 2 *IsNewVersionOf* version 1 |
+| Compiles / IsCompiledBy | When code or workflows produce outputs | An analysis script *Compiles* a figure |
+
+For a complete list, see [DataCite's relatedIdentifier documentation](https://support.datacite.org/docs/datacite-metadata-schema-v44-recommended-and-optional-properties#102-relatedidentifier).
+
+### How to Add Related Identifiers
+
+The specific process depends on which repository you're using, but the general approach is similar:
+
+**During initial deposit:**
```

Comment:
All of the example of bold text like this should really be headers. It is much better for navigation and accessibility.